server/src/assembly/resources/conf/iotdb-env.sh

```diff
@@ -66,19 +66,19 @@ calculate_heap_sizes()
     fi
 
     # set max heap size based on the following
-    # max(min(1/2 ram, 1024MB), min(1/4 ram, 8GB))
+    # max(min(1/2 ram, 1024MB), min(1/4 ram, 64GB))
     # calculate 1/2 ram and cap to 1024MB
```

Comment:
Should this be changed?
```suggestion
    # calculate 1/2 ram and cap to 8192MB
```